packages/validation/docs/configuration.md

```diff
@@ -186,13 +186,32 @@ settings:
 
 ### Resource Links
 
-Validates whether the comply with a JSON schema.
+Validates whether links/references between resources are valid - i.e. if the target
+object exists or not. 
 
-There are no settings for this plugin.
+This plugin has one setting to configure if missing optional links should be seen
+as failures or not - default is false.
 
 ```yaml
 plugins:
   resource-links: true
+settings:
+  resource-links:
+    fail-missing-optional: true
```

Comment:
Not sure whether this would be better as a separate rule with level as fail basically indicates "error"? I guess this would require a bit more work so we can use this for now.

Maybe there are better rule names but something like:
```
rules:
  resource-links/no-missing-required: "err"
  resource-links/no-missing-optionals: "err" // fail or "warn"
```